TAC/Meetings/2022-04-21/CCC_TAC_Minutes-2022-04-21.md

```diff
@@ -0,0 +1,190 @@
+* **Recordings**: [YouTube Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
+
+### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* **Accenture** - Giuseppe Giordano
+* **Ant Group** - Zongmin Gu
+* **Arm** - Thomas Fossati / Michael Lu
+* **Google** - Iulia Ion
+* **Huawei** - Zhipeng (Howard) Huang
+* **Intel** - Dan Middleton / Simon Johnson
+* **Meta** - Eric Northup / Shankaran Gnanashanmugam
+* **Microsoft** - Dave Thaler (TAC chair)
+* **Red Hat** - Lily Sturmann / Dimitrios Pendarakis
+
+## Links
+
+* **Code of Conduct**: [code-of-conduct.confidentialcomputing.io](https://code-of-conduct.confidentialcomputing.io)
+* **CCC Charter**: [charter.confidentialcomputing.io](https://charter.confidentialcomputing.io)
+* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+
+Rep. Thaler opened the call at 7:05am PT. He welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation.
+
+### Roll Call / Attendance
+
+#### Voting members
+
+* Dave Thaler (Microsoft) P
+* Lily Sturman (Red Hat) P
+* Simon Johnson (Intel) P
+* Zhipeng (Howard) Huang (Huawei) P
+* Thomas Fossati (Arm) P
+
+
+#### Attendees
+
+* Eric Voit (Cisco) P 
+* Penglin Yang (China Mobile) P
+* Dan Middleton (Intel) P
+* Steve Van Lare (Anjuna) P
+* Jethro Beekman (Fortanix) P
+* Henk Birkholz (Fraunhofer-Gesellschaft) P
+* Vijay Nayani (Huawei) P
+* Xinxin Fan (IoTeX) P
+* Sampo Sovio (Huawei) P
+* Ravi Sahita (Rivos) P
+
+
+#### Also Participating
+* Kurt Taylor (Linux Foundation) P
+* Shubhra Kar (Linux Foundation) P      
+
+### Meeting minute approvals
+
+#### Approved by voting members
+
+**Resolved:** That the minutes of the [April 07, 2022](../2022-04-07/TAC_Minutes-2022-04-07.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
```

Comment:
```suggestion
**Proposed:** That the minutes of the [April 07, 2022](../2022-04-07/TAC_Minutes-2022-04-07.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
```
I agree with Lily that the text above is confusing when it was not in fact approved, and I think it should be fixed in these minutes.  It would also be fine to remove this sentence and instead link to the minutes from line 82.

Reply:
This was inherited from Brian, I suspect that it is a rules of order thing, but I will be happy to reword it.